src/modules/kvs/test/commit.c

```diff
@@ -1434,23 +1372,11 @@ void commit_process_big_fileval (void) {
     /* This root is
      *
      * root_ref
-     * "dir" : dirref to dir_ref
-     *
-     * dir_ref
      * "val" : val w/ "42"
-     *
      */
 
-    dir = treeobj_create_dir ();
-    treeobj_insert_entry (dir, "val", treeobj_create_val ("42", 2));
-
-    ok (kvs_util_json_hash ("sha1", dir, dir_ref) == 0,
-        "kvs_util_json_hash worked");
-
-    cache_insert (cache, dir_ref, cache_entry_create (dir));
-
     root = treeobj_create_dir ();
-    treeobj_insert_entry (root, "dir", treeobj_create_dirref (dir_ref));
+    treeobj_insert_entry (root, "val", treeobj_create_val ("42", 2));
```

Comment:
As a whole in the tests I'm not bothering to check for NULL on things like ```treeobj_create_dir()``` or ```cache_entry_create()``` or similar, as I figure malloc errors are unlikely and checks will just pollute the test files with way more code.